packages/vscode-extension/README.md

```diff
@@ -1,131 +1,92 @@
-# Microsoft 365 Agents Toolkit for Visual Studio Code
+# Microsoft 365 Agents Toolkit - Previously known as Teams Toolkit
```

Comment:
Don't need this previously note

Reply:
Could you please suggest why? this note is to ensure a smooth transit for users who used to Teams Toolkit and working on it already.

Reply:
I resolved it per doc team suggestion: an evolution of Teams Toolkit